packages/react-components/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-components",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "clean": "shx rm -rf dist/",
+    "precompile": "npm run clean",
+    "compile": "tsc",
+    "lint": "eslint src"
+  },
+  "devDependencies": {
+    "@mui/icons-material": "^5.15.10",
+    "@mui/material": "^5.15.10",
+    "@pagopa/pagopa-editorial-components": "^2.1.4",
```

Comment:
Why is this dependency here?

I though the purpose of react-components was to detach from editorial-components altogether.

Reply:
It will be an incremental migration. The idea is that we can keep the components coming from editorial-components (at the moment) as long as they do not give us headaches. If we are having troubles with some component, then we should migrate them to the `react-components` package and change the references of those components.

I agree with you with the purpose of the new package: we are just moving in that direction one step at a time

Reply:
That's all clear to me, but it would equate to having editorial-components as a dependency inside nextjs-website wouldn't it?

I'm still not sure on why we need it as a dependency for react-components at all.

Unless we plan on moving the non-problematic components inside react-components while still importing them from editorial-components and just wrapping them with styling and props parsing.

Reply:
that's right. the final goal would be to have all the react components within the `react-components` library, so the `nextjs-website` will depend only on `react-components` without knowing anything about `@pagopa/pagopa-editorial-components`.
The `react-components` package should handle all the "headaches" (which also mean to move a component from editorial components library to this package).

I should migrate also the `Accordion` component that at the moment is located under the `nextjs-website` app to the `react-components` package, but I haven't done that just because I want to validate the idea before.
If this is going to be accepted, then IMHO we should keep ALL the React components under the new `react-components` package and remove the `@pagopa/pagopa-editorial-components` dependency from the `nextjs-website` app (and in future remove this dependency for good).

Is it a little bit clearer now the vision of these changes?

Reply:
Yup, sounds good!